OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -23,7 +23,10 @@ public class DrawLineToTargetInfo : ITraitInfo
 		public readonly int Delay = 60;
 
 		[Desc("Width (in pixels) of the target lines.")]
-		public readonly int LineWidth = 2;
+		public readonly int LineWidth = 1;
+
+		[Desc("Width (in pixels) of the target lines.")]
+		public readonly int QueuedLineWidth = 1;
 
 		[Desc("Width (in pixels) of the end node markers.")]
 		public readonly int MarkerWidth = 3;
```

Comment:
Can we reset this back to 2, too? 3:1 looks a bit odd

Reply:
OK. Done.